packages/next/test/unit/utils.test.ts

```diff
@@ -54,6 +54,7 @@ describe('getImagesConfig', () => {
       domains: ['example.com'],
       sizes: [512, 1024],
       remotePatterns: undefined,
+      localPatterns: [{ search: '' }],
```

Comment:
Might be good to have one with `pathname`

Reply:
Added in [d770e23](https://github.com/vercel/vercel/pull/12195/commits/d770e2379f0ac66162d88e107cd0aebb58a2eb0c)